client/src/styles/ArtDetails.css

```diff
@@ -0,0 +1,82 @@
+.modal-artDetails {
+  position: fixed;
+  left: 50%;
+  transform: translateX(-50%);
+  bottom: 6rem;
+  width: 98%;
+  background-color: var(--fourth-color);
+  border: 2.5px solid var(--fifth-color);
+  border-radius: 15px;
+  box-shadow: 0 5px 15px rgba(0, 0, 0, 0.5);
+  z-index: 1;
+  overflow: scroll;
+  scrollbar-color: var(--third-color) var(--third-color);
+}
+
+.modal-artDetails > header {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.close-artDetails {
+  font-family: var(--primary-font);
+  font-size: 1.5rem;
+  color: var(--secondary-color);
+  background-color: transparent;
+  border: none;
+  padding-right: 1rem;
+  margin-top: 1rem;
+  margin-bottom: 1rem;
+}
+
+.close-artDetails:hover,
+.close-artDetails:focus {
+  color: var(--third-color);
+  cursor: pointer;
+}
+
+.modal-content-artDetails {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 2rem;
+  padding-left: 1.5rem;
```

Comment:
You could also change this one with : "padding: 0 1.5rem"

Reply:
Done